roles/ucp/tasks/main.yml

```diff
@@ -32,6 +32,10 @@
     - "{{ ucp_instance_id_file }}"
   when: node_name != ucp_bootstrap_node_name
 
+- name: copy the ucp license to the remote machine
+  copy: src={{ ucp_license_file }} dest={{ ucp_license_remote }}
```

Comment:
@vvb shall we set the default value for `ucp_license_file` to be empty string and then conditionally run this task only when user provides a license file?

I am thinking we need to continue to support unlicensed deploy of ucp and it won't be possible with the current change as it needs `ucp_license_file` to be defined to a valid file, right?